### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by Eatonasher** [MicrosoftDocs/visualstudio-docs#9519]
I'm trying to follow this tutorial, but when I try to build the "Test" application following the instructions precisely, I get the error: "The debug executable ....GettingStarted\WalkthroughFirstWPFApp\csharp\bin\x64\Test\net7.0-windows\ExpenseIt9.exe' specified in the 'ExpenseItIntro' debug profile does not exist" - any ideas?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: a5db8407-cc91-c741-6bc4-9d059ab40308
* Version Independent ID: 8d477def-5203-6085-915b-31971afeccab
* Content: [Tutorial: Build an application - Visual Studio (Windows)](https://learn.microsoft.com/en-us/visualstudio/ide/walkthrough-building-an-application?view=vs-2019)
* Content Source: [docs/ide/walkthrough-building-an-application.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/ide/walkthrough-building-an-application.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-compile**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by TimShererWithAquent** [comment]
Hi @Eatonasher - I don't know how to troubleshoot your configuration. Here's a couple of resources:

- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  

Here's a couple of pages I found that might refer to the error you describe:

- https://learn.microsoft.com/en-us/answers/questions/750023/debug-profile-does-not-exist-how-to-get-it
- https://stackoverflow.com/questions/61140481/visual-studio-debug-executable-specified-in-the-debug-profile-does-not-exist

I'm going to keep this issue open as a doc enhancement in case there's a fix we should make to the article to avoid this. If you get it working, please let us know!

**Comment by Eatonasher** [comment]
After manually changing the target framework from 7.0 to 5.0 in the project file, I was able to recreate the error simply by following the instructions again. This time, I was able to open up the "ExpenseItIntro2" properties dialogue and see that the "Target Framework" dropdown is no longer greyed out (it was greyed out as ".NET 7.0" before I manually changed it and reloaded the project)

![image](https://github.com/MicrosoftDocs/visualstudio-docs/assets/9275121/25621dac-e840-4d3c-9710-b2f8cbfeaffe)

This shows that the assembly name is "ExpenseIt9", which was in the error message previously. Does this shed any light on the problem? 
Thank you

**Comment by ghogen** [comment]
@Eatonasher 
I just went through the tutorial and reproduced the steps (except the last part about specifying a server share), and I was able to get through each step without error. This is with a recent VS 2022 build.

The error about the .NET SDK seems concerning. Check your "Output" tab and see if the project actually built successfully. It could be that you have an installation problem.  You might try uninstalling either or both VS and any .NET SDKs and reinstalling them.
